src/TicTacToe.elm

```diff
@@ -11,31 +12,42 @@ main =
 
 -- MODEL
 
-type alias Model = Game
+type PlayState = NotStarted | InProgress Game
```

Comment:
Nice :+1: These types make elm code really easy to read for someone who's not familiar with the codebase. Explicit good, magic bad.